src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,57 @@
+package calculator;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Calculator {
+
+	private final Separator separator;
+
+	public Calculator(Separator separator) {
+		this.separator = separator;
+	}
+
+	public int splitAndSum(String targetValue) {
+		if (isNullOrEmpty(targetValue)) {
+			return 0;
+		}
+
+		List<String> splitValues = separator.splitByUnits(targetValue);
+		List<Integer> splitNumbers = parseValuesAsNumber(splitValues);
+
+		return sumNumber(splitNumbers);
+	}
+
+	private boolean isNullOrEmpty(String value) {
+		return (value == null || value.isEmpty());
+	}
+
+	private List<Integer> parseValuesAsNumber(List<String> values) {
+		return values.stream().map(this::parseValueAsNumber).collect(Collectors.toList());
+	}
+
+	private int parseValueAsNumber(String value) {
+		try {
+			int number = Integer.parseInt(value);
+			validateNumberIsNegative(number);
+			return number;
+		} catch (NumberFormatException exception) {
+			throw new RuntimeException("숫자 이외의 값을 입력하셨습니다.");
+		}
+	}
+
+	private void validateNumberIsNegative(int number) {
+		if (isNumberNegative(number)) {
+			throw new RuntimeException("음수를 입력하셨습니다.");
+		}
+	}
+
+	private boolean isNumberNegative(int number) {
+		return (number < 0);
```

Comment:
굳이 괄호를 적어둘 필요는 없을거 같아요 :) 
```suggestion
		return number < 0;
```

Reply:
괄호를 감싸지 않는 것에 대해 별도의 이유가 있나요?

저는 조건식에 대해서 괄호를 감싸는 것이, 조건을 분명히 할 수 있다고 생각하고 있으며
그렇게 시간이 흐르다보니 습관화 되어 버렸습니다.

혹시나 이 습관이 별로 좋지 않다고 지적하신다면, 곧바로 고쳐나가도록 하겠습니다.